### ISSUE TITLE
Building Campania from Fontforge

### ISSUE THREAD

**Issue by ecstrema** [MarcSabatella/Campania#12]
I am trying to use campania as a base to make a ChordSymbols font.
If you don't mind, here's a little question: do you build campania from FontForge? If yes, then where are the substitutions loaded from?
In the "font information" dialog, I see only a list of substitution and no file is referenced.

Thank you

**Comment by ecstrema** [comment]
I also just saw that for his figurato font, FKretlow used a fontforge script, did you use it too?

**Comment by fkretlow** [comment]
If you alter the feature files you need to re-merge them from within Fontforge before you build the fonts manually. The command is in the File menu.

A Python script can speed the whole build process up, especially when you need to do a lot of fine tuning in the feature files. It’s pretty straightforward, I have one in the Figurato repository if you want to take a look.

**Comment by ecstrema** [comment]
> 
> 
> If you alter the feature files you need to re-merge them from within Fontforge before you build the fonts manually. The command is in the File menu.
> 
> A Python script can speed the whole build process up, especially when you need to do a lot of fine tuning in the feature files. It’s pretty straightforward, I have one in the Figurato repository if you want to take a look.

Thank you! It looks perfect!

**Comment by MarcSabatella** [comment]
Indeed, I load the feature file into FontForge manually using the menu.  It's kind of a pain, because as far as I know every time you make a change you need to first go to the font properties and delete all the old tables (the word "merge" in the menu item doesn't really seem to apply).

Anyhow, I too have thought about adapting this for chord symbols but haven't gotten around to it, I think it's a great idea if you'd like to try!  And I can certainly help.

**Comment by ecstrema** [comment]
Thank you for the help and sorry for not responding. Fontforge's python API is quite incredible actually and I was able to fully automate this. However, I stopped making a chord symbol font because I made a simple JS renderer instead, which was all I needed.
But I've since then used fontforge api for other fonts, including [saxy](https://github.com/Marr11317/Saxy) and if you ever want to automate that behaviour, that script python script in saxy's repo should do all the job. There's also instructions on how to use it in the [docs folder](https://github.com/Marr11317/Saxy/blob/master/docs/contributing.md)
